src/app/+visualization/co-author-network/co-author-network.component.html

```diff
@@ -50,7 +50,7 @@
         <div class="card">
           <div class="card-header">{{ 'VISUALIZATION.NETWORK.COAUTHOR.COPARTICIPANT' | translate }}</div>
           <div class="card-body">
-            <table class="table table-sm">
+            <table class="table table-sm" aria-labelledby="coparticipantTable">
```

Comment:
The same as the above comment about `objectsPerYearTable`. 

In fact, it looks like all of the `table` cases are doing something like that.